antsibull/write_docs.py

```diff
@@ -315,14 +334,14 @@ async def output_indexes(collection_info: CollectionInfoT,
         collection_toplevel = dest_dir
 
     async with asyncio_pool.AioPool(size=lib_ctx.thread_max) as pool:
-        for collection_name, plugin_maps in collection_info.items():
+        for collection_name, plugin_maps in collection_to_plugin_info.items():
             if not squash_hierarchy:
                 collection_dir = os.path.join(collection_toplevel, *(collection_name.split('.')))
             else:
                 collection_dir = collection_toplevel
             writers.append(await pool.spawn(
                 write_plugin_lists(collection_name, plugin_maps, collection_plugins_tmpl,
-                                   collection_dir)))
+                                   collection_dir, collection_metadata.get(collection_name))))
```

Comment:
Is there a reason to use get instead of `collection_metadata[collection_name]` here?